### PR TITLE
MAINT: update CI for mystery-stew location

### PR DESCRIPTION
**Pull request by ebolyen**

### CODE REVIEW

File: q2galaxy/core/environment.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def iter_deps(self, *packages, include_self=True, _seen=None):
 
         for package in packages:
             for dependency in self.iter_primary_deps(package):
-                if dependency in _seen:
+                # __glibc isn't a "real" package, there may be others like it
+                if dependency in _seen or dependency.startswith('__'):
```
Comment (thermokarst) [inline q2galaxy/core/environment.py:51]: Here's how I handled this over in genome-sampler:

https://github.com/caporaso-lab/genome-sampler/pull/96/files

Comment (ebolyen) [reply q2galaxy/core/environment.py:51]: Do you think we can rely on this:

> Other virtual packages will be added in future conda releases. These are denoted by a leading double-underscore in the package name.

I've never seen a real conda package named that way, which is probably why they went with that as their naming convention

Comment (ebolyen) [reply q2galaxy/core/environment.py]: Looks like there's already other ones not listed:
```
§ conda info

     active environment : None
       user config file : /home/evan/.condarc
 populated config files : 
          conda version : 4.9.1
    conda-build version : 3.21.3
         python version : 3.8.5.final.0
       virtual packages : __glibc=2.26=0
                          __unix=0=0
                          __archspec=1=x86_64
```

Comment (ebolyen) [reply q2galaxy/core/environment.py]: this is actually super cool, I like this concept a lot, it'll make it possible to have more specialized packages that the resolver can actually understand

**Pull request merged**